src/main/java/frc/robot/subsystems/intake/BeamBreaker.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+
+public class BeamBreaker extends LoggedSubsystem<BeamBreakerLoggedInputs> {
+    public DigitalInput beam = new DigitalInput(Ports.Intake.BEAMBREAKER);
+    boolean isBlocking;
+
+    public BeamBreaker(BeamBreakerLoggedInputs inputs) {
+        super(inputs);
+    }
+
+
+    public boolean getBeam() {
+        return beam.get();
+    }
```

Comment:
More descriptive!!! Does this method mean that the beam is connected or disconnected?

Reply:
done.